source/extensions/filters/common/lua/lua.h

```diff
@@ -69,6 +69,19 @@ namespace Lua {
   lua_pushnumber(state, val);                                                                      \
   lua_settable(state, -3);
 
+/**
+ * Get absl::string_view from Lua string. This checks if the argument at index is a string
+ * and build an absl::string_view from it.
+ * @param state the current Lua state.
+ * @param index the index of argument.
+ * @return absl::string_view of Lua string with proper string length.
+ **/
+inline absl::string_view getStringViewFromLuaString(lua_State* state, int arg) {
+  size_t input_size = 0;
+  const char* input = luaL_checklstring(state, arg, &input_size);
```

Comment:
According to @dio 's comments, we also need to add a comment here to explain the case of parameter errors (the case that `luaL_checklstring` cannot get valid string).